ml-unit-test-client/build.gradle

```diff
@@ -64,4 +64,49 @@ if (project.hasProperty("myBintrayUser")) {
 	}
 }
 
-tasks.test.dependsOn mlDeploy
\ No newline at end of file
+task createHttpCredentials(type: com.marklogic.gradle.task.MarkLogicTask) {
+  def client = getAppConfig().newAppServicesDatabaseClient("Security")
+  String xquery = """
+xquery version "1.0-ml";
+
+import module namespace sec = "http://marklogic.com/xdmp/security"
+      at "/MarkLogic/security.xqy";
+
+declare option xdmp:mapping "false";
+
+try {
+  sec:remove-credential("ml-unit-test-credentials")
```

Comment:
Does the Manage PUT endpoint not work if the credential already exists?

Reply:
Unfortunately, the REST endpoint only supports AWS authentication according to the documentation.